src/serving.ts

```diff
@@ -129,55 +106,22 @@ export class Server {
       res.status(200).send('ok');
     });
 
-    this.app.post('/check', (req, res) => {
-      if(!req.body) {
-        // TODO: don't thow error, return an appropriate response code.
-        throw Error('No request body.');
-      }
-
-      this.log.write(`Request: ${JSON.stringify({headers: req.rawHeaders, body: req.body})}`);
-
-      // The request format translation below exists so that we restrict
-      // queries to our website server to hitting the attribute specified
-      // in our config (TOXICITY).
-      // TODO(rachelrosen): Consider a cleaner way to do this, such as
-      // using booleans, to avoid the extra translation code.
-
-      let requestData: AnalyzeCommentData = req.body as AnalyzeCommentData;
-
-      let requestedAttributes: RequestedAttributes = {};
-      requestedAttributes[this.config.toxicityAttribute] = {
-        score_type: 'PROBABILITY'
-      };
-
-      let context: Context|undefined = undefined;
-      if (requestData.articleText) {
-        context = {
-          article_and_parent_comment: {
-            article: { text: requestData.articleText},
-          }
-        };
-        if (context.article_and_parent_comment &&
-            requestData.parentComment) {
-          context.article_and_parent_comment.parent_comment = {
-            text: requestData.parentComment
-          };
-        }
-      }
+    // Define an endpoint with CORS enabled to be used by the plugin.
+    this.app.options('/demo/check', cors()) // enable CORS for pre-flight requests.
+    this.app.post('/demo/check', cors(), (req, res) => {
```

Comment:
If you specify cores in the options for a URL prefix, you should not need it in the post option also.

Reply:
I just tried it, looks like it is required for both preflight and actual POST request.

Reply:
Oh! OK, I stand corrected! Good to know!